scripts/utils/verify_scripts.js

```diff
@@ -110,19 +109,19 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
   ) {
     const log = logActivated ? (...a) => console.log(...a) : () => {}
 
-    const BatchExchangeArtifact = require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange")
-    const networkId = await web3.eth.net.getId()
-    const BatchExchange = new web3.eth.Contract(BatchExchangeArtifact.abi, BatchExchangeArtifact.networks[networkId].address)
-
-    const auctionElementsDecoded = await getOrdersPaginated(BatchExchange, pageSize)
     const bracketTraderAddresses = brackets.map((address) => address.toLowerCase())
-
-    // Fetch all token infos(decimals, symbols etc) and prices upfront for the following verification
-    const relevantOrders = auctionElementsDecoded.filter((order) => bracketTraderAddresses.includes(order.user.toLowerCase()))
     const BatchExchangeContract = Contract(require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange"))
     BatchExchangeContract.setProvider(web3.currentProvider)
     const exchange = await BatchExchangeContract.deployed()
 
+    // Fetch all token infos(decimals, symbols etc) and prices upfront for the following verification
+    const ordersObjects = await Promise.all(
+      bracketTraderAddresses.map(async (bracketAddress) =>
+        exchangeUtils.decodeOrders(await exchange.getEncodedUserOrders.call(bracketAddress))
```

Comment:
It appears the internal call `await exchange.getEncodedUserOrders.call(bracketAddress)` is returning null and `decodeOrders` can't handle it. I will look into the contracts repo about handling this.

Reply:
Ok, so it turns out your problem is here orders object is `[ undefined ]` which is then looped over later. There was an import statement which corrected the error with `decodeOrders`, but then later you are trying to access `order.sellToken` which wont work when `order` is `undefined`. 

I think that the construction of `orderObjects` here should be revisited.

I will push my change to the import statements.